### DISCUSSION TITLE
Data Scientist

### DISCUSSION THREAD

**Discussion by agogStrider** [WeAreMahsaAmini/Core#215] [Introductions]
Hi,
I can help with data science/analytics projects. I also have experience in ML/DL specially in NLP, but open to other fields as well.
Best,

**Comment by ehsaaanes** [comment]
Hi
I'm looking for people who can help creating an app for donation. 
I’d appreciate it if you check it here #163. 
please write a comment or let me know what you think about it. 
Too much thanks*
